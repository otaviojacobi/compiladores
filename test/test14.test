int main () {

    float *a = (float*)malloc(sizeof(float)*10);
    float a[0] = 12.+0e2;
    float a[1] = 3.-78e+13;
    float a[2] = 3.78e+13;
    float a[3] = -3.-3;

    return a;
}