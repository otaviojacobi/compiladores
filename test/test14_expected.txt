1 TK_PR_INT [int]
1 TK_IDENTIFICADOR [main]
1 TK_ESPECIAL [(]
1 TK_ESPECIAL [)]
1 TK_ESPECIAL [{]
3 TK_PR_FLOAT [float]
3 TK_ESPECIAL [*]
3 TK_IDENTIFICADOR [a]
3 TK_ESPECIAL [=]
3 TK_ESPECIAL [(]
3 TK_PR_FLOAT [float]
3 TK_ESPECIAL [*]
3 TK_ESPECIAL [)]
3 TK_IDENTIFICADOR [malloc]
3 TK_ESPECIAL [(]
3 TK_IDENTIFICADOR [sizeof]
3 TK_ESPECIAL [(]
3 TK_PR_FLOAT [float]
3 TK_ESPECIAL [)]
3 TK_ESPECIAL [*]
3 TK_LIT_INT [10]
3 TK_ESPECIAL [)]
3 TK_ESPECIAL [;]
4 TK_PR_FLOAT [float]
4 TK_IDENTIFICADOR [a]
4 TK_ESPECIAL [[]
4 TK_LIT_INT [0]
4 TK_ESPECIAL []]
4 TK_ESPECIAL [=]
4 TK_LIT_INT [12]
4 TK_ESPECIAL [.]
4 TK_LIT_INT [+0]
4 TK_IDENTIFICADOR [e2]
4 TK_ESPECIAL [;]
5 TK_PR_FLOAT [float]
5 TK_IDENTIFICADOR [a]
5 TK_ESPECIAL [[]
5 TK_LIT_INT [1]
5 TK_ESPECIAL []]
5 TK_ESPECIAL [=]
5 TK_LIT_INT [3]
5 TK_ESPECIAL [.]
5 TK_LIT_INT [-78]
5 TK_IDENTIFICADOR [e]
5 TK_LIT_INT [+13]
5 TK_ESPECIAL [;]
6 TK_PR_FLOAT [float]
6 TK_IDENTIFICADOR [a]
6 TK_ESPECIAL [[]
6 TK_LIT_INT [2]
6 TK_ESPECIAL []]
6 TK_ESPECIAL [=]
6 TK_LIT_FLOAT [3.78e+13]
6 TK_ESPECIAL [;]
7 TK_PR_FLOAT [float]
7 TK_IDENTIFICADOR [a]
7 TK_ESPECIAL [[]
7 TK_LIT_INT [3]
7 TK_ESPECIAL []]
7 TK_ESPECIAL [=]
7 TK_LIT_INT [-3]
7 TK_ESPECIAL [.]
7 TK_LIT_INT [-3]
7 TK_ESPECIAL [;]
9 TK_PR_RETURN [return]
9 TK_IDENTIFICADOR [a]
9 TK_ESPECIAL [;]
10 TK_ESPECIAL [}]
